docs/contribution/40-how-to/5-support.mdx

```diff
@@ -2,11 +2,15 @@
 title: How to Get Support for Extension Development
 ---
 
-We use the [mittwald/contributor-support](https://github.com/mittwald/contributor-support) Github project to exchange information between mittwald and its contributors. You can send questions and feature requests to mittwald there, contact other contributors, and submit a new extension or edit an existing one via an issue. Please note that this repository (and all issues therein) is public.
+As soon as you are listed as a contributor, you get access to a [private GitHub repository](https://github.com/mittwald/contributor-support) where all contributors and mittwald exchange information.
+You can send questions and feature requests to mittwald there, contact other contributors, and submit a new extension or edit an existing one via an issue.
 
 We also use that GitHub repository to announce important changes that concern extensions.
 
-For private issues, such as a question regarding billing, you can ask our customer service via the mStudio ticket system.
+If you have personal matters, such as a question regarding billing, you can ask our customer service via the mStudio ticket system.
 
 mittwald would not be mittwald if we didn't value the importance of direct communication next to the also important indirect path of communication.
 Let us know what you need support with and we will put the technical contact person on the call or in the Zoom call.
+
+We support you daily from Monday to Friday in all aspects of contribution.
+However, not on public holidays in NRW.
```

Comment:
I concur that this is unnecessarily detailed. I would just write something unspecific in the sense of "during regular business hours", and then refer to the "Erreichbarkeit" section of https://www.mittwald.de/impressum.